src/core/data/sagas/createOrReplaceAssociation.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+import {
+  call,
+  put,
+  takeEvery,
+} from '@redux-saga/core/effects';
+import { DataApiActions, DataApiSagas } from 'lattice-sagas';
+import { Logger } from 'lattice-utils';
+import type { Saga } from '@redux-saga/core';
+import type { WorkerResponse } from 'lattice-sagas';
+import type { SequenceAction } from 'redux-reqseq';
+
+import { deleteEntitiesWorker } from './deleteEntities';
+
+import { CREATE_OR_REPLACE_ASSOCIATION, createOrReplaceAssociation, deleteEntities } from '../actions';
+
+const { createAssociations } = DataApiActions;
+const { createAssociationsWorker } = DataApiSagas;
+
+const LOG = new Logger('DataSagas');
+
+function* createOrReplaceAssociationWorker(action :SequenceAction) :Saga<WorkerResponse> {
+
+  const workerResponse :Object = {};
+  const { id } = action;
```

Comment:
You're only using `id` on line 28, and you're using `action.id` everywhere else. Just delete this line and use `action.id`.